packages/volto/src/config/validation.ts

```diff
@@ -150,6 +151,13 @@ const registerValidators = (config: ConfigType) => {
     dependencies: { behaviorName: 'plone.eventbasic', fieldName: 'end' },
     method: endEventDateRangeValidator,
   });
+
+  config.registerUtility({
+    name: 'default_language',
+    type: 'validator',
+    dependencies: { format: 'default_language' },
```

Comment:
Can you use `fieldName: 'default_language'` so https://github.com/plone/plone.i18n/pull/73 isn't needed?

Reply:
We don't have this feature (yet). Do you really think it's the way to go? For me, the problem is the lack of expressivity in the definition of the fields in the backend source, which is not Volto to blame. And it's not only here. It's all over the place in the backend fields definitions.